.github/workflow_scripts/gsprocessing_pytest.sh

```diff
@@ -0,0 +1,4 @@
+cd /usr/lib/spark/graphstorm/graphstorm-processing/
+pip install .
```

Comment:
Let's move this to be pip3 as well to ensure we're installing to the right python env.